doc/migration/Iron.rst

```diff
@@ -1,50 +1,112 @@
 Iron to Jazzy
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-component parser
-*****************
-Changes from `(PR #1256) <https://github.com/ros-controls/ros2_control/pull/1256>`__
-
-* All ``joints`` defined in the ``<ros2_control>``-tag have to be present in the URDF received :ref:`by the controller manager <doc/ros2_control/controller_manager/doc/userdoc:subscribers>`, otherwise a ``std::runtime_error`` is thrown. This is to ensure that the URDF and the ``<ros2_control>``-tag are consistent. E.g., for configuration ports use ``gpio`` tags instead.
-* The syntax for mimic joints is changed to the `official URDF specification <https://wiki.ros.org/urdf/XML/joint>`__. The parameters within the ``ros2_control`` tag are not supported any more. Instead of
-
-  .. code-block:: xml
-
-    <ros2_control name="GazeboSystem" type="system">
-      <joint name="right_finger_joint">
-        <command_interface name="position"/>
-        <state_interface name="position">
-          <param name="initial_value">0.15</param>
-        </state_interface>
-        <state_interface name="velocity"/>
-        <state_interface name="effort"/>
+General
+*******
+* A ``version.h`` file will be generated per package using the ament_generate_version_header  (`#1449 <https://github.com/ros-controls/ros2_control/issues/1449>`_). For example, you can include the version of the package in the logs.
+
+  .. code-block:: cpp
+
+    #include <controller_manager/version.h>
+    ...
+    RCLCPP_INFO(get_logger(), "controller_manager version: %s", CONTROLLER_MANAGER_VERSION_STR);
+
+controller_interface
+********************
+For details see the controller_manager section.
+
+* Pass URDF to controllers on init (`#1088 <https://github.com/ros-controls/ros2_control/issues/1088>`_).
+* Pass controller manager update rate on the init of the controller interface  (`#1141 <https://github.com/ros-controls/ros2_control/issues/1141>`_)
+* A method to get node options to setup the controller node #api-breaking (`#1169 <https://github.com/ros-controls/ros2_control/issues/1169>`_)
```

Comment:
```suggestion
* Pass URDF to controllers on init (`#1088 <https://github.com/ros-controls/ros2_control/pull/1088>`_).
* Pass controller manager update rate on the init of the controller interface  (`#1141 <https://github.com/ros-controls/ros2_control/pull/1141>`_)
* A method to get node options to setup the controller node #api-breaking (`#1169 <https://github.com/ros-controls/ros2_control/pull/1169>`_)
```

Reply:
That's funny, I just copied this from the auto-generated release notes.